themebuilder-scss/package.json

```diff
@@ -19,7 +19,9 @@
     "semver": "^5.6.0"
   },
   "scripts": {
-    "test": "jest --coverage --verbose --detectOpenHandles",
+    "test-code": "jest --coverage --verbose --detectOpenHandles",
+    "lint": "eslint --config .eslintrc-themebuilder .",
```

Comment:
It looks like `"root": true` helps us to simplify this command to:
```"lint": "eslint ."```

Reply:
there is no problem to run 'eslint .' in themebuilder-scss folder. The problem - run 'eslint .' in the root of the repository. Eslint parse .eslintrc in themebuilder-scss folder (yes, it parse .eslintrc file in folder that we include in .eslintignore). So we need to rename .eslintrc in something other.

Reply:
It looks strange to me:
- how VS Code will process the ThemeBuilder config?
- why root ESLint parses `.eslintrc` in an ignored folder?

Let's ask ESLint devs about this issue? :)

Reply:
Eslint really parses all .eslintrc, even in .eslintignore https://github.com/eslint/eslint/issues/13147